### PR TITLE
Information about bank synchronization with Enable Banking.

### PR DESCRIPTION
**Pull request by oskarenablebanking**
Hi. Im oskar from Enable Banking. We are being contacted often by Odoo users with questions about issues / authentication with Enable Banking. Im making this pull request so things would be clearer to Odoo users. Could this also be added to master?
Is there something I should improve?

### CODE REVIEW

File: content/applications/finance/accounting/bank/feeds/enablebanking.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+     
+#. After giving consent for sharing data with Odoo, you may be prompted with your bank credentials, or redirected to your bank for authorization.
+
+#. Post authorization, you should be redirected to Odoo
+
+FAQ
+===
+
+I have an error with authorization.
+-------------------------------------------------------------------
```
Comment (FlorianGilbert) [inline content/applications/finance/accounting/bank/feeds/enablebanking.rst:46]:
```suggestion
I have an error with authorization
-------------------------------------------------
```

File: content/applications/finance/accounting/bank/feeds/enablebanking.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Additional authorization guidance currently available for
+
+Finnish banks:
+https://tilisy.enablebanking.com/guides/FI/
+
+Swedish banks:
+https://tilisy.enablebanking.com/guides/SE/
+
+Im seeing duplicate transactions.
+-------------------------------------------------------------------
```
Comment (FlorianGilbert) [inline content/applications/finance/accounting/bank/feeds/enablebanking.rst:61]:
```suggestion
Im seeing duplicate transactions
-----------------------------------------------
```
Probably we have to change the name of this part

**Review by FlorianGilbert — CHANGES_REQUESTED** [review]
Not a real review, need to have a review from doc team

**Comment by chiaraprattico** [comment]
Hi @oskarenablebanking, thanks for your contribution. I'm putting this in draft as I will be checking about the content and form of your PR.

**Review by oskarenablebanking — COMMENTED** [review]
updated

**Comment by chiaraprattico** [comment]
Hi @oskarenablebanking, I took your work and started a new branch to finalize the content and upload it online. So I'll close your branch, you can see updates here https://github.com/odoo/documentation/pull/3858.

Thanks!

**Pull request closed without merging**